### PR TITLE
[OPM] Add tests to check for report consistency

### PR DESCRIPTION
**Pull request by esoergel**
@czue @sravfeyn
I wasn't actually able to find an edge case that's clearly causing this issue, but here's the test code I was using.

### CODE REVIEW

File: custom/opm/tests/case_reports.py [hunk 1]
```diff
@@ -136,3 +141,45 @@ def test_mock_data(self):
...
         )
         row = MockCaseRow(case, report)
+
+
+def make_case_row(form_props=None, vhnd_props=None):
+    """
+    Accepts lists of properties available in form and at the vhnd
+    and returns a corresponding case row
+    """
+    report_year= 2014
```
Comment (dimagimon) [inline custom/opm/tests/case_reports.py]: E225 missing whitespace around operator

File: custom/opm/tests/test_consistency.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+from datetime import date, datetime
```
Comment (dimagimon) [inline custom/opm/tests/test_consistency.py]: F401 'date' imported but unused

Comment (dimagimon) [inline custom/opm/tests/test_consistency.py]: F401 'datetime' imported but unused

File: custom/opm/tests/test_consistency.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+from couchforms.models import XFormInstance
```
Comment (dimagimon) [inline custom/opm/tests/test_consistency.py]: F401 'XFormInstance' imported but unused

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: custom/opm/tests/case_reports.py [hunk 4]
```diff
@@ -7,11 +7,11 @@
 
 from casexml.apps.case.models import CommCareCase
 from couchforms.models import XFormInstance
-from ..reports import SharedDataProvider
 from dimagi.utils.dates import DateSpan, add_months
 
 from ..beneficiary import OPMCaseRow
-from ..reports import CaseReportMixin
+from ..constants import *
```
Comment (millerdev) [inline custom/opm/tests/case_reports.py]: was this to bring in `CFU1_XMLNS`? is it doing more that that?

**Comment by millerdev** [comment]
I want @dimagimon to complain about `import *` so I don't have to.

File: custom/opm/tests/case_reports.py (hunk 4 above)
Comment (czue) [inline custom/opm/tests/case_reports.py]: i agree import * is not great here. referencing things like `constants.CFU1_XMLNS` is a nice workaround

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: custom/opm/tests/case_reports.py (hunk 4 above)
Comment (czue) [inline custom/opm/tests/case_reports.py]: @millerdev updated

File: custom/opm/tests/test_consistency.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
+from ..constants import *
+from .case_reports import *
```
Comment (millerdev) [inline custom/opm/tests/test_consistency.py]: Missed these two. My fault for not flagging them before.

Comment (czue) [inline custom/opm/tests/test_consistency.py]: @esoergel

**Pull request merged**